modules/swagger-codegen/src/main/resources/JavaPlayFramework/newApi.mustache

```diff
@@ -3,11 +3,12 @@ package {{package}};
 {{#imports}}import {{import}};
 {{/imports}}
 
-import play.mvc.Http;
```

Comment:
Here too we can't remove this because if we have a `Http.MultipartFormData.FilePart` somewhere in the file, it will not work.